components/cluster/pacemaker/Makefile

```diff
@@ -32,13 +32,18 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+PREFERRED_BITS = 64
+PYTHON_VERSION = 3.5
+PYTHON_VENDOR_PACKAGES=$(PYTHON.$(PYTHON_VERSION).VENDOR_PACKAGES.64)
+PYTHON =  $(PYTHON.$(PYTHON_VERSION).$(BITS))
```

Comment:
I think `PREFERRED_BITS` are required only when `32_and_64` bits are being build and one wants 64-bit binaries in `/usr/bin/` instead of `/usr/bin/64/` (and even then the variable should be before `include ../../../make-rules/shared-macros.mk` because it alters things there).

Isn't `PYTHON_VERSION=3.5` enough? Latter `PYTHON` lines look verbatim from `make-rules/shared-macros.mk`.

Reply:
Indeed. Just use PYTHON_VERSION = 3.5 is enough.